pkg/cable/driver.go

```diff
@@ -71,7 +71,11 @@ func AddDriver(name string, driverCreate DriverCreateFunc) {
 }
 
 // Returns a new driver according the required Backend
-func NewDriver(localEndpoint types.SubmarinerEndpoint, localCluster types.SubmarinerCluster) (Driver, error) {
+func NewDriver(localEndpoint *types.SubmarinerEndpoint, localCluster *types.SubmarinerCluster) (Driver, error) {
+	if localCluster == nil || localEndpoint == nil {
+		return nil, fmt.Errorf("NewDriver called with nil localCluster and/or localEndpoint")
+	}
```

Comment:
I assume you missed removing the nil checks here as was done elsewhere.

```suggestion
```

Reply:
Indeed, thanks!